src/app/cart/services/cart.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+import { ITovar } from '../../spare-parts/models/tovar.model';
+import { ITotalCart } from "../models/total-card.model";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CartService {
+  items: ITotalCart[] = [];
+  totalInCart = 0;
+
+  eventChangedCountTovar = new Subject<number>();
```

Comment:
if you set in variable Subject or Behavior Subject or any other stream you should to celled variable with "$" (for example: eventChangedCountTovar$)